project_template/tasks.py

```diff
@@ -298,8 +298,30 @@ class TaskOwnedIncomeFromSalariesTable(CountTableRowsTask):
     child_class = None
 
 
+class TaskOwnedBuildingRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedBuildingsSingleRowEntry
+    template_name = "tasks/owned_buildings.html"
+
+    def save_verified_data(self, verified_data):
+        owner_person, created = models.Person.objects.get_or_create(
+            name=verified_data['owner_name'],
+            surname=verified_data['owner_surname']
+        )
+
+        owned_building, created = models.OwnedBuildingsTableEntry.objects.get_or_create(
```

Comment:
Two issues here:
1. you aren't filling in the `observations` column in the model with any of the information provided in the form. If you write `observations=""`, then anything that the user might have written in the form is thrown away and your model will be populated with an empty string.
2. you forgot to populate the `owner_person` field with the foreign key from the `owner_person` model you just instantiated and populated above.

Reply:
Regarding the `owner_person` foreign key, it's supposed to populate a field called `coowner`, according to the model in `models.py`.